### PR TITLE
Created a lambda module and used the module to spin up the lambda in …

### PR DESCRIPTION
**Pull request by chubbymilo**
- Created a module for the Python lambda
- Created a mock test environment that spins up the lambda in a different region

### CODE REVIEW

File: lambda/variables.tf [hunk 1]
```diff
@@ -20,6 +19,11 @@ variable "lambda_file_extention" {
   }
 }
 
+variable "lambda_source_file_path" {
```
Comment (chubbymilo) [inline lambda/variables.tf:22]: Added this path to point to the directory that has the app.py file

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
+terraform {
```
Comment (ryan-caarter) [inline main.tf:1]: Put required_version in the one terraform declaration?

File: test-region/main.tf [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "5.50.0"
+    }
+  }
+}
+
```
Comment (ryan-caarter) [inline test-region/main.tf:13]: Declare the aws provider explicitly here and you'll be able to pass a different region in for each main.tf 👍

File: lambda/variables.tf (hunk 1 above)
Comment (chubbymilo) [reply lambda/variables.tf:22]: I will merge the PR and created new one for using S3 to store states

File: test-region/main.tf [hunk 4]
```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_version = ">= 1.2.0"
```
Comment (ryan-caarter) [inline test-region/main.tf:2]: Don't commit the lambda.zip file, this is generated on terraform plan 😎

**Pull request merged**